source/api/discovery/0.5/index.md

```diff
@@ -82,127 +82,138 @@ The key words _MUST_, _MUST NOT_, _REQUIRED_, _SHALL_, _SHALL NOT_, _SHOULD_, _S
 
-A complete change history is not required, and sometimes not even desirable. If there are many, frequent changes to a resource, then an implementation _MAY_ omit any number of individual changes, but _SHOULD_ always  have the most recent change included in the list. Changes that are deemed to be insignificant to the publisher of the list _MAY_ also be omitted, such as changes that affect only the syntax of the document but not the content.
+A complete change history is not required, and sometimes not even desirable. If there are many or frequent changes to a resource, an implementation _MAY_ omit any number of individual changes, but _SHOULD_ always  have the most recent change included in the list. Changes that are deemed to be insignificant to the publisher of the list _MAY_ also be omitted, such as changes that affect only the syntax of the document but not the content.
 
 Example Level 2 Activity:
 
-```
-{
-  "type": "Create",
-  "object": {
-    "id": "https://example.org/iiif/1/manifest",
-    "type": "Manifest"
+```json-doc
+[
+  {
+    "type": "Create",
+    "object": {
+      "id": "https://example.org/iiif/1/manifest",
+      "type": "Manifest"
+    },
+    "endTime": "2017-09-20T00:00:00Z"
   },
-  "endTime": "2017-09-20T00:00:00Z"
-}
+  // ...
+]
 ```
 
 #### 2.1.4. Aggregating Activities
 {: #aggregated-activity-streams}
 
-Activities can be added to or removed from streams at times other than when the resources they refer to are created, modified, deleted or moved. There are several use cases for this pattern, however they are all outside of the normal publication of a stream by the content owners and thus are unlikely to affect most publishing implementations. They do affect consuming applications, and the implications are documented in the section on processing streams.
+Activities can be added to or removed at times other than when the resources they refer to are created, modified, deleted or moved. There are several use cases for this pattern, however they are all outside of the normal publication of a stream by the content owners and thus are unlikely to affect most publishing implementations. They do affect consuming applications, and the implications are documented in [3.5. Activity Streams Processing Algorithm][discovery-processing-streams].
 
-The first use case is when third-party aggregators add activities harvested from different institutions, and likely filtered according to some criteria, into a composite stream. For example, a dinosaur aggregator might filter several natural history museum streams for only those manifests that are about dinosaurs.
+The first use case is when third-party aggregators add activities harvested from different institutions, filter those activities or objects, and aggregate them into a composite stream. For example, a dinosaur aggregator might filter several natural history museum streams for only those manifests that are about dinosaurs.
```

Comment:
Mostly :+1:  The aggregator might not filter at all, but I don't think it makes the description invalid or even unlikely. It's a motivating use case, and the filtering is more probable than not.